app/controllers/paging_schedule_controller.rb

```diff
@@ -6,10 +6,13 @@ class PagingScheduleController < ApplicationController
 
   def show
     schedule = PagingSchedule.for(request_for_schedule)
```

Comment:
I'm happy with this, but is not having a paging schedule exceptional (and we could rescue the exception with a 404..)?